From 431aa9f30348f4724a87f65ec07d5949dbe0b97f Mon Sep 17 00:00:00 2001
From: Alexey Minnekhanov <alexeymin@postmarketos.org>
Date: Mon, 6 Oct 2025 16:27:50 +0300
Subject: [PATCH] HACK dts: sdm660: Disable all GPU freqs above nominal
 for stability

Remove all GPU clock frequences that require power domain levels
above nominal to make GPU less crashy. This results in less frequent
hangchecks and more stability.

Signed-off-by: Alexey Minnekhanov <alexeymin@postmarketos.org>

Source-Commit: https://github.com/sdm660-mainline/linux/commit/431aa9f30348f4724a87f65ec07d5949dbe0b97f
Scope: SDM660-common
Porting notes: Retain the original stable nominal-frequency cap after 0020 deletes the inherited SDM630 OPP table and reconstructs SDM660's table.
---
diff --git a/arch/arm64/boot/dts/qcom/sdm660.dtsi b/arch/arm64/boot/dts/qcom/sdm660.dtsi
index fa9c6f596537a2..70505591f2d651 100644
--- a/arch/arm64/boot/dts/qcom/sdm660.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm660.dtsi
@@ -74,20 +74,20 @@
 		}; */
 
 		/* bins 157,146 -> bits 6,5 */
-		opp-700000000 {
+		/*opp-700000000 {
 			opp-hz = /bits/ 64 <700000000>;
 			opp-level = <RPM_SMD_LEVEL_TURBO>;
 			opp-peak-kBps = <5184000>;
 			opp-supported-hw = <0x60>;
-		};
+		};*/
 
 		/* bins 157,146,135 -> bits 6,5,4 */
-		opp-647000000 {
+		/*opp-647000000 {
 			opp-hz = /bits/ 64 <647000000>;
 			opp-level = <RPM_SMD_LEVEL_NOM_PLUS>;
 			opp-peak-kBps = <4068000>;
 			opp-supported-hw = <0x70>;
-		};
+		};*/
 
 		/* bins 157,146,135 -> bits 6,5,4 */
 		opp-588000000 {
